pkg/arguments/interactive.go

```diff
@@ -295,12 +295,14 @@ func PromptIPNet(fs *pflag.FlagSet, flagName string) error {
 		if flag.Changed {
 			return nil
 		}
-
+		// If the default value is nil, set the value as a default value
+		if flag.DefValue == "<nil>" {
+			flag.DefValue = flag.Value.String()
+		}
```

Comment:
It's not clear to me why we need to set the flag default value here, can you explain, please?

Reply:
We set the default value so that it is set (and shown) on the console when the user does not input any values. When we set the value in https://github.com/openshift-online/ocm-cli/pull/552/files#diff-2f19822c49dc0bec88a73920ea0f04a30b7047b5a827c90447e02b7378744f97R627, it sets the "value" attribute and not the defValue attribute.

Reply:
This is the implementation of the // TODO below, right. You can probably move that comment here - it will be more clear.

Reply:
Moved